### PR TITLE
Fractal: new, 7

### PR DESCRIPTION
**Pull request by OriginCode**
Topic Description
-----------------

- pango: update to 1.52.2
    Signed-off-by: Kaiyang Wu <origincode@aosc.io>
- fractal: new, 7
    Signed-off-by: Kaiyang Wu <origincode@aosc.io>
- libshumate: new, 1.2.1
    Signed-off-by: Kaiyang Wu <origincode@aosc.io>

Package(s) Affected
-------------------

- fractal: 7
- libshumate: 1.2.1
- pango: 1.52.2

Security Update?
----------------

No

Build Order
-----------

```
#buildit pango libshumate fractal
```

Test Build(s) Done
------------------

**Primary Architectures**

- [x] AMD64 `amd64`
- [x] AArch64 `arm64`
- [x] LoongArch 64-bit `loongarch64`

**Secondary Architectures**

- [ ] Loongson 3 `loongson3`
- [x] PowerPC 64-bit (Little Endian) `ppc64el`
- [x] RISC-V 64-bit `riscv64`

**Experimental Architectures**

- [ ] MIPS R6 64-bit (Little Endian) `mips64r6el`

### CODE REVIEW

**Comment by OriginCode** [comment]
LGTM on amd64:
圖片

File: desktop-gnome/fractal/autobuild/defines [hunk 1]
```diff
@@ -0,0 +1,14 @@
+PKGNAME=fractal
+PKGSEC=gnome
+PKGDEP="glib gtk-4 libadwaita gstreamer gtksourceview-5 openssl libshumate \
+    sqlite pipewire pango graphene"
+BUILDDEP="meson ninja rustc llvm xdg-desktop-portal"
+PKGDES="A Matrix messaging app for GNOME"
```
Comment (MingcongBai) [inline desktop-gnome/fractal/autobuild/defines]: A Matrix messaging client for GNOME

File: desktop-gnome/fractal/spec [hunk 2]
```diff
@@ -0,0 +1,5 @@
+VER=7.0
+# FIXME: APT does not recognize single digit version
```
Comment (MingcongBai) [inline desktop-gnome/fractal/spec]: single digit version without REL *

File: desktop-gnome/libshumate/autobuild/defines [hunk 3]
```diff
@@ -0,0 +1,13 @@
+PKGNAME=libshumate
+PKGSEC=gnome
+PKGDEP="glib gtk-4 gobject-introspection cairo sqlite libsoup json-glib \
+    protobuf-c sysprof"
+BUILDDEP="meson ninja gi-docgen gperf vala"
+PKGDES="A GTK toolkit providing widgets for embedded maps"
```
Comment (MingcongBai) [inline desktop-gnome/libshumate/autobuild/defines]: GTK toolkit widgets for embedded maps

**Review by MingcongBai — CHANGES_REQUESTED** [review]

**Review by MingcongBai — APPROVED** [review]

**Pull request merged**